unidist/config/backends/mpi/envvars.py

```diff
@@ -47,3 +47,23 @@ class MpiLog(EnvironmentVariable, type=bool):
 
     default = False
     varname = "UNIDIST_MPI_LOG"
+
+
+class MpiSharedObjectStore(EnvironmentVariable, type=bool):
+    """Whether to enable shared object store or not."""
+
+    default = True
```

Comment:
```suggestion
    default = False
```
We decided to use the shared memory as an option to opt in for now.